packages/spatial/src/physics/classes/Physics.ts

```diff
@@ -382,6 +383,7 @@ function updateRigidbodyPose(entities: Entity[]) {
     if (!world) continue
     const body = world.Rigidbodies.get(entity)
     if (!body) continue
+    if (hasComponent(entity, NetworkObjectComponent) && !hasComponent(entity, NetworkObjectAuthorityTag)) continue
```

Comment:
Can we add a test case for this around here? 
https://github.com/ir-engine/ir-engine/blob/139a060ca21b7055f6cf96bc3aab0eff56dab0bb/packages/spatial/src/physics/classes/Physics.test.tsx#L1191

> should not do anything when the entity has a NetworkObjectComponent and it does not have a NetworkObjectAuthorityTag

Reply:
Good point